operator/controllers/seldondeployment_controller.go

```diff
@@ -938,7 +938,7 @@ func createDeploymentWithoutEngine(depName string, seldonId string, seldonPodSpe
 					Annotations: map[string]string{},
 				},
 			},
-			Strategy: appsv1.DeploymentStrategy{RollingUpdate: &appsv1.RollingUpdateDeployment{MaxUnavailable: &intstr.IntOrString{StrVal: "10%"}}},
+			Strategy: appsv1.DeploymentStrategy{Type: appsv1.RecreateDeploymentStrategyType},
```

Comment:
Why we need to update the DeploymentStrategy?

Reply:
Because the old pod will be deleted when the new one is running now. 
This rule will lead to excedquota of resource.